components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/Util.java

```diff
@@ -96,4 +98,9 @@ public static ClaimHandler mockClaimHandler() throws FrameworkException {
                 any(Map.class), anyBoolean());
         return claimHandler;
     }
+
+    public static void mockIdentityUtil(){
+        mockStatic(IdentityUtil.class);
+        when(IdentityUtil.getLocalGroupsClaimURI()).thenReturn(UserCoreConstants.ROLE_CLAIM);
```

Comment:
can we specify why this is needed?

Reply:
To address the issue of java.lang.ExceptionInInitializerError related to the IdentityUtil.getLocalGroupsClaimURI()